token-cache/cloud-run.yaml

```diff
@@ -0,0 +1,14 @@
+name: app-name
+memory: 256Mi
+cpu: 200m
+min-instances: 1
+
+platform:
+  gke:
+    connectivity: external
+    domain-mappings:
+      prod:
+        - app-name.retailsvc.com
+      staging:
+        - app-name.retailsvc.com
+    opa-enabled: false
```

Comment:
Will this be deployed? If it will please set the min-instance to 0 and fix the name and URLs

Reply:
No, will remove it